lib/express_validator.js

```diff
@@ -99,6 +99,19 @@ var expressValidator = function(options) {
     return this;
   };
 
+  ValidatorChain.prototype.withMessage = function(message) {
+    if (!!this.lastError) {
+      var error = this.validationErrors.pop();
```

Comment:
Is there a reason we define error here? I would think it only needs to be defined the second time.